app/src/main/java/woowacourse/movie/activity/TicketResultActivity.kt

```diff
@@ -26,41 +19,44 @@ class TicketResultActivity : AppCompatActivity() {
     }
 
     private fun initTicketDataView() {
-        val info: TicketingInfo = intent.customGetSerializable(INFO_KEY)
-        initTitle(info.title)
-        initPlayingDate(info.playingDate, info.playingTime)
-        initCount(info.count)
-        initPrice(info.price, info.count)
-        initPricePayment(info.payment)
+        val ticketModel: TicketModel = getReceivedIntentData()
+        initTitle(ticketModel.reservationInfoModel.title)
+        initPlayingDate(
+            ticketModel.reservationInfoModel.playingDate,
+            ticketModel.reservationInfoModel.playingTime
+        )
+        initCountAndSeat(ticketModel.reservationInfoModel.count, ticketModel.seats)
+        initPricePayment(ticketModel.price, ticketModel.reservationInfoModel.payment)
     }
 
+    private fun getReceivedIntentData(): TicketModel = intent.customGetSerializable(TICKET_KEY)
+
     private fun initTitle(title: String) {
         val titleView = findViewById<TextView>(R.id.text_title)
         titleView.text = title
     }
 
-    private fun initPlayingDate(playingDate: LocalDate, playingTime: LocalTime) {
+    private fun initPlayingDate(playingDate: String, playingTime: String) {
         val playingDateView = findViewById<TextView>(R.id.text_playing_date)
         playingDateView.text = getString(
             R.string.date_time,
-            DateFormatter.format(playingDate),
-            TimeFormatter.format(playingTime)
+            playingDate,
+            playingTime
         )
     }
 
-    private fun initCount(count: Int) {
-        val countView = findViewById<TextView>(R.id.text_person_count)
-        countView.text = count.toString()
-    }
-
-    private fun initPrice(price: Price, count: Int) {
-        val priceView = findViewById<TextView>(R.id.text_price)
-        priceView.text = DecimalFormatter.format(price.price * count)
+    private fun initCountAndSeat(count: Int, seats: List<SeatModel>) {
+        val countView = findViewById<TextView>(R.id.text_count_seat)
+        countView.text = getString(
+            R.string.count_seat_info,
+            count,
+            seats.joinToString(", ") { it.row + it.column }
```

Comment:
joinToString의 기본 separator값이 ", "이라서 생략해도 됩니다 🙂 
```suggestion
            seats.joinToString { it.row + it.column }
```